tests/README.md

```diff
@@ -1,5 +1,51 @@
 This directory contains the end-to-end tests of prplMesh
 
+## API for writing tests
+
+Several helper classes are available that abstract the entities involved in the tests.
+The test environment prepares these before the tests are started.
+
+- `env.controller`, `env.agents[0]` and `env.agents[1]` are `ALEntity` objects with the following attributes:
+    - `mac`: the AL-MAC address of the entity, to be used e.g. when sending a CMDU to it.
+    - `radios`: on agents only, list of radios on this agent (see below).
```

Comment:
Not sure if relevant to this commit,
but the mapping between radios and log files (e.g. `beerocks_agent_wlan2.log`) is not clear to me.

Reply:
Good point. I added that to the wait_for_log explanation.